packages/browser/src/core/analytics/index.ts

```diff
@@ -109,6 +111,13 @@ function _stub(this: never) {
   console.warn(deprecationWarning)
 }
 
+const popPageContext = (args: any[]): PageContext | undefined => {
```

Comment:
`unknown[]` would be work, but its the same level of typechecking. I am choosing to not make this because this final argument is "hidden" from typescript, as the way that positional arguments work here (see how these Params types are actually made to understand) make it impossible to safely add an "any last argument" (which I actually tried). The actual type signature is insane, and can't even be represented with function overloading AFAIK.